src/pages/admin/store/pickup/[uuid].tsx

```diff
@@ -0,0 +1,106 @@
+import { PickupOrdersDisplay } from '@/components/admin/store';
+import { PickupEventStatus } from '@/components/admin/store/PickupEventCard';
+import { Typography } from '@/components/common';
+import { EventCard } from '@/components/events';
+import { config } from '@/lib';
+import { StoreAPI } from '@/lib/api';
+import withAccessType from '@/lib/hoc/withAccessType';
+import { CookieService, PermissionService } from '@/lib/services';
+import { PublicOrderPickupEvent } from '@/lib/types/apiResponses';
+import { CookieType } from '@/lib/types/enums';
+import { formatEventDate } from '@/lib/utils';
+import styles from '@/styles/pages/StorePickupEventDetailsPage.module.scss';
+import { GetServerSideProps } from 'next';
+import Link from 'next/link';
+import { useState } from 'react';
+
+interface PickupEventDetailsPageProps {
+  pickupEvent: PublicOrderPickupEvent;
+}
+
+const PickupEventDetailsPage = ({ pickupEvent }: PickupEventDetailsPageProps) => {
+  const { status, title, start, end, orderLimit, description, linkedEvent, orders } = pickupEvent;
+  const [ordersView, setOrdersView] = useState<'fulfill' | 'prepare'>('fulfill');
+
+  return (
+    <div className={styles.page}>
+      <Link href={config.admin.store.pickup} className={styles.back}>
+        {'< Back to dashboard'}
+      </Link>
+      <div className={styles.container}>
+        <div className={styles.pickupDetails}>
+          {linkedEvent ? (
+            <EventCard event={linkedEvent} attended={false} hideInfo />
+          ) : (
+            <Typography variant="h3/regular" className={styles.noEvent}>
+              *Not linked to an event on the membership portal
+            </Typography>
```

Comment:
I think you could just hide the event card if there's no linked event. when it's red it looks like an error even though it isn't really one

Reply:
To be honest I like it better when it looks kinda error-y because I think we should strongly encourage having a linked event for pickup events going forward since it'll be missing a cover image if not.